### ISSUE TITLE
Request: jump to start/end of current block

### ISSUE THREAD

**Issue by jkelleyrtp** [DnailZ/oomotion_vscode#8]
It'd be nice if there was a way to jump to the start or end of the current block, and progressively jump upwards or downwards.

**Comment by Richiban** [comment]
You can try [CodeFlea](https://marketplace.visualstudio.com/items?itemName=Richibansoft.codeflea) for this. I have both extensions installed; they play nicely together. You can have CodeFlea keybindings that only apply in the Oomotion NORMAL modes.

**Comment by DnailZ** [comment]
So what do you mean by a `block`? In rust, it's simple that it is something inside a curly brace `{...}`. I just don't have the idea where should we jump when we are using python.

**Comment by Richiban** [comment]
To be honest, line-tree mode is probably what the OP wants. @jkelleyrtp you should try it out!

**Comment by well1791** [comment]
@Richiban do you still use oomotion together with codeflea?

**Comment by Richiban** [comment]
> @Richiban do you still use oomotion together with codeflea?

No, not since CodeFlea went modal. Now CodeFlea and OOMotion occupy the same space and it doesn't make much sense to use both.
